updater/script/test

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
```

Comment:
If we went the route of suggesting devs run tests within the `docker-dev-shell`, then maybe this should be renamed to `updater/script/ci-test` to be consistent with the scripts used in the ecosystem folders?